cfme/tests/automate/test_customization_paginator.py

```diff
@@ -49,8 +55,7 @@ def test_paginator(some_dialogs, soft_assert):
         for text in get_relevant_rows(dialogs_table):
             dialogs_found.add(text)
         current_rec_offset = paginator.rec_offset()
-    soft_assert(dialogs_found == set(map(lambda dlg: dlg.label, some_dialogs)),
-                "Could not find all dialogs by clicking the paginator!")
+    soft_assert(set([dlg.label for dlg in some_dialogs]).issubset(dialogs_found))
```

Comment:
This should just be `assert set([dlg.label for dlg in some_dialogs]).issubset(dialogs_found)`, since it's the last line of the test.

Was the "helpful" assertion error message removed for a reason?